### DISCUSSION TITLE
How to do css nesting and referring to other classes?

### DISCUSSION THREAD

**Discussion by peterbartels** [chakra-ui/panda#725] [Q&A] [answered]
Would it possible to do something like this for example? I tried but it didn't work:

```
const child= css({
  color: 'green.400',
})

const parent= css({
  [child]: {
    color: 'red.400',
  },
});

```

**Comment by anubra266** [comment, marked as answer]
@peterbartels panda generates atomic classNames, so don't do that.
Just target the child within the parent styles with any selector of your choice.
